rpc/src/v1/helpers/mod.rs

```diff
@@ -34,8 +34,10 @@ mod signer;
 mod signing_queue;
 mod subscribers;
 mod subscription_manager;
+mod verify_signature;
```

Comment:
this file doesn't exist change it to `mod signature` or change the file name itself?